app/src/main/java/com/example/grutor/Fragments/LessonsFragment.java

```diff
@@ -29,6 +41,64 @@ public View onCreateView(LayoutInflater inflater, ViewGroup parent, Bundle saved
     public void onViewCreated(View view, Bundle savedInstanceState) {
         // Setup any handles to view objects here
         // EditText etFoo = (EditText) view.findViewById(R.id.etFoo);
+
+        fab_main = view.findViewById(R.id.fab);
+        fabCamera = view.findViewById(R.id.fabCamera);
+        fabMessages = view.findViewById(R.id.fabMessages);
+        fab_close = AnimationUtils.loadAnimation(getApplicationContext(), R.anim.fab_close);
+        fab_open = AnimationUtils.loadAnimation(getApplicationContext(), R.anim.fab_open);
+        fab_clock = AnimationUtils.loadAnimation(getApplicationContext(), R.anim.fab_rotate_clock);
+        fab_anticlock = AnimationUtils.loadAnimation(getApplicationContext(), R.anim.fab_rotate_anticlock);
```

Comment:
if these variables are only used in this functions, they aren't need to be defined to class level variables.